switches/http/example-nodered-simple.json

```diff
@@ -0,0 +1,126 @@
+[
```

Comment:
I would put examples codes in a subdirectory, and test code in another subdirectory. Alternatively, and maybe better, we should put example code in the top-level `example/` directory and test code in the top-level `tests/` directory. But I understand that because of dependencies, you might want to keep asebahttp stuff bundled together.